server/runtime/src/main/server/bin/service/infinispan.service

```diff
@@ -0,0 +1,18 @@
+# script is for /etc/systemd/system
+# activate with 'systemctl daemon-reload'
+#               'systemctl start|enable infinispan'
+
+[Unit]
+Description=Infinispan Server Service
+After=network.target
+
+[Service]
+Environment="INFINISPAN_HOME=/opt/infinispan"
+#Environment="JAVA_HOME=/usr/java/jdk-11.0.8"
+ExecStart=/bin/bash -c ${INFINISPAN_HOME}/bin/server.sh
+Type=simple
+User=infinispan
```

Comment:
I'm not very familiar with how systemd starts user services, would it work if we removed the `User=infinispan` line and instead copy the service file to `/etc/systemd/user/infinispan`?

Reply:
Hi @danberindei 
if you use systemd/user (https://wiki.archlinux.org/index.php/Systemd/User) this will start the process with user permission, but ONLY if the user is active.
For ISPN this is not what you want, the process should start if the system is up and running (or by manual systemctl ...) with the user permission, but independent from the user activity